app/main.dev.ts

```diff
@@ -17,8 +17,15 @@ import path from 'path';
 import { app, BrowserWindow } from 'electron';
 import { autoUpdater } from 'electron-updater';
 import log from 'electron-log';
+import windowState from 'electron-window-state';
+
 import MenuBuilder from './menu';
 
+// CSP enabled in production mode, don't warn in development
+delete process.env.ELECTRON_ENABLE_SECURITY_WARNINGS;
+process.env.ELECTRON_DISABLE_SECURITY_WARNINGS = 'true';
```

Comment:
Is there a reason we have to `delete` before re-assigning?

Reply:
It's a different env var, enable/disable. This is just a hack to get rid of the warning in development.